tests/cli/performance_test.py

```diff
@@ -0,0 +1,204 @@
+
+        # Verify it actually returns results
+        assert len(results) > 0, "Autocompletion should return at least some results"
+
+        # Verify wurzel built-in steps are included
+        assert any("wurzel.steps." in r for r in results), "Should include wurzel built-in steps"
+
+    def test_autocompletion_performance_user_project(self):
+        """Test that autocompletion is fast in a user project directory with custom steps."""
+        with tempfile.TemporaryDirectory() as temp_dir:
+            temp_path = Path(temp_dir)
+
+            # Create a user project structure with custom steps
+            steps_dir = temp_path / "mysteps"
+            steps_dir.mkdir()
+
+            # Create a custom TypedStep file
+            custom_step_file = steps_dir / "custom_step.py"
+            custom_step_file.write_text("""
+from wurzel.step import TypedStep
+
+class MyCustomStep(TypedStep):
+    pass
+
+class AnotherStep(TypedStep):
+    pass
+""")
+
+            # Change to the user project directory
+            import os
+
+            original_cwd = os.getcwd()
+            try:
+                os.chdir(temp_path)
+
+                # Test performance from user project
+                times = []
+                for _ in range(3):
+                    start = time.perf_counter()
+                    results = complete_step_import("")
+                    end = time.perf_counter()
+                    duration = end - start
+                    times.append(duration)
+
+                avg_time = sum(times) / len(times)
+                max_time = max(times)
+
+                # Assert fast performance even with user project scanning
+                assert avg_time < 1.0, f"User project autocompletion too slow: {avg_time:.3f}s average (expected < 1.0s)"
+                assert max_time < 2.0, f"User project autocompletion max time too slow: {max_time:.3f}s (expected < 2.0s)"
+
+                # Verify it finds both wurzel steps and user steps
+                assert len(results) > 0, "Should find some steps"
+                assert any("wurzel.steps." in r for r in results), "Should include wurzel built-in steps"
+                assert any("mysteps." in r for r in results), "Should include user-defined steps"
+
+            finally:
+                os.chdir(original_cwd)
+
+    def test_autocompletion_performance_with_prefix(self):
+        """Test that autocompletion with prefix filtering is fast."""
+        times = []
+        for _ in range(3):
+            start = time.perf_counter()
+            results = complete_step_import("wurzel.steps.")
+            end = time.perf_counter()
+            duration = end - start
+            times.append(duration)
+
+        avg_time = sum(times) / len(times)
+
+        # Should be fast since we're filtering
+        assert avg_time < 0.5, f"Prefix autocompletion too slow: {avg_time:.3f}s average (expected < 0.5s)"
```

Comment:
```suggestion
        assert avg_time < 0.5, f"Prefix autocompletion too slow: {avg_time:.3f}s average (expected < 0.7s)"
```
Fix for windows runner perfromance